website_form_builder/README.rst

```diff
@@ -0,0 +1,106 @@
+.. image:: https://img.shields.io/badge/license-LGPL--3-blue.svg
+   :target: https://www.gnu.org/licenses/lgpl
+   :alt: License: LGPL-3
+
+====================
+Website Form Builder
+====================
+
+This module provides websites the feature of adding custom forms in any page.
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+#. Have *Administration / Settings* privileges.
+#. Go to *Settings > Activate developer mode*.
+#. Go to *Settings > Technical > Database Structure > Models*.
+#. Search for the model you want to manage website form access for.
+#. When you find it, it will have a *Website Forms* section where you can:
+
+   * Allow the model to get forms, by checking *Allowed to use in forms*.
+   * Give the model forms a better name in *Label for form action*.
+   * Choose the field where to store custom fields data in *Field for custom
+     form data*. If you leave this one empty and the model is a mail thread,
+     a new message will be appended with that custom data.
+
+#. In the *Fields* tab, there's a new column called *Blacklisted in web forms*.
+   It's a security feature that forbids form submitters to write to those
+   fields. When you create a new website form, all its model fields are
+   automatically whitelisted for the sake of improving the UX. If you want to
+   have higher control, come back here after creating the form and blacklist
+   any fields you want.
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to any of your website pages.
+#. Edit it.
+#. Drag and drop the *Form* snippet into the page.
+#. Use the snippet overlay to add, edit and remove fields.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/186/10.0
+
+Known issues / Roadmap
+======================
+
+* These type of fields will not appear, they are forbidden since they make no
+  sense in this module's context, or a correct implementation would be adding
+  not much value while adding lots of complexity:
+
+  * ``id``
```

Comment:
[That happens automatically](https://github.com/odoo/odoo/blob/052323fae4b8f37428e844a80cb807c9eefbc47c/addons/website_form/models/models.py#L57), I was just trying to avoid having to fill the docs gap from `website_form`, but I think I'll have to do it after all 😆